creedengophp.properties

```diff
@@ -1,12 +1,20 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for PHP language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-php
-publicVersions=2.0.0
-archivedVersions=
+publicVersions=2.0.1
+archivedVersions=2.0.0
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-php-plugin
 
+2.0.1.description=improve library versions; check and upgrades for SonarQube until 25.3.0 version
+2.0.1.sqcb=[24.12,LASTEST]
```

Comment:
You got this right in https://github.com/SonarSource/sonar-update-center-properties/pull/687/files
Please use it as your versions reference. 

Also note spelling mistake: "`LASTEST`" vs `LATEST`

Reply:
Hi @ganncamp, it's done.
I corrected only the mistake. I don't understand if you are waiting another thing.